templates/taxi/manufacturer_form.html

```diff
@@ -0,0 +1,11 @@
+{% extends 'base.html' %}
+
+{% block content %}
+  <h1>{{ object|yesno:"Update, Create" }} driver</h1>

```

Comment:
The heading text incorrectly refers to 'driver' instead of 'manufacturer'. It should be '{{ object|yesno:"Update, Create" }} manufacturer' to accurately reflect the form's purpose.